wandb/proto/wandb_internal_pb2.py

```diff
@@ -8,3 +8,9 @@
     from wandb.proto.v4.wandb_internal_pb2 import *
 elif protobuf_version == "5":
     from wandb.proto.v5.wandb_internal_pb2 import *
+else:
+    raise ImportError(
+        "Failed to import protobufs for protobuf version"
+        f" {google.protobuf.__version__}. `wandb` only works with major"
```

Comment:
the example in the pr description looks kinda weird then... why 5.27.1 doesn't work?

Reply:
5.27.1 does work---I produced the sample by removing the other cases in the if-else chain. This is just to test what happens when the `ImportError` line is hit.